pkgs/tools/X11/wmutils-opt/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, libxcb }:
+
+stdenv.mkDerivation rec {
+  name = "wmutils-opt-${version}";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/wmutils/opt/archive/v${version}.tar.gz";
+    sha256 = "0mfhpja2l592i2a3j2l0lg65jhsdw1b57jnv2l0759ngpl4v82zk";
+  };
+
+  buildInputs = [ libxcb ];
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
```

Comment:
Then, could you please add yourself as a maintainer in the `meta`? (Don't forget to also add a line to `lib/maintainers.nix`.)